src/Setup/CLI/ReplaceResponsiveLogoCommand.php

```diff
@@ -0,0 +1,106 @@
+<?php declare(strict_types=1);
+
+/* Copyright (c) 2022 Thibeau Fuhrer <thibeau@sr.solutions> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\Setup\CLI;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use DirectoryIterator;
+
+/**
+ * @author Thibeau Fuhrer <thibeau@sr.solutions>
+ */
+class ReplaceResponsiveLogoCommand extends Command
+{
+    protected const RESPONSIVE_LOGO_PATH = '/images/HeaderIconResponsive.svg';
+    protected const COMMON_LOGO_PATH = '/images/HeaderIcon.svg';
+
+    protected static $defaultName = 'replace-responsive-logo';
+    protected ?string $delos_responsive_logo_hash;
+    protected ?string $delos_common_logo_hash;
+    protected string $delos_path;
+    protected string $ilias_path;
```

Comment:
Typed properties are not supported in PHP 7.3, which we must support in ILIAS 7.

Reply:
Thanks for the review!